docs/source/tutorials/yolox.md

```diff
@@ -122,28 +141,36 @@ python tools/export.py configs/detection/yolox/yolox_s_8xb16_300e_coco.py \
 ```
 
 ### Inference
+Download exported models([preprocess](http://pai-vision-data-hz.oss-accelerate.aliyuncs.com/EasyCV/modelzoo/detection/yolox/yolox-pai/model/export/epoch_300_pre_notrt.pt.preprocess), [model](http://pai-vision-data-hz.oss-accelerate.aliyuncs.com/EasyCV/modelzoo/detection/yolox/yolox-pai/model/export/epoch_300_pre_notrt.pt.blade), [meta](http://pai-vision-data-hz.oss-accelerate.aliyuncs.com/EasyCV/modelzoo/detection/yolox/yolox-pai/model/export/epoch_300_pre_notrt.pt.blade.config.json)) or export your own model.
+Put them in the following format:
+```shell
+export_blade/
+??? epoch_300_pre_notrt.pt.blade
```

Comment:
??? should be replaced

Reply:
sorry for the mistake, and have fixed it.